Cubical/Algebra/CommRing/Localisation/Base.agda

```diff
@@ -76,27 +77,29 @@ module Loc (R' : CommRing {ℓ}) (S' : ℙ (R' .fst)) (SMultClosedSubset : isMul
  locSym : isSym _≈_
  locSym (r , s , s∈S') (r' , s' , s'∈S') (u , p) = u , sym p
 
+ open VarNames5 R'
  locTrans : isTrans _≈_
  locTrans (r , s , s∈S') (r' , s' , s'∈S') (r'' , s'' , s''∈S') ((u , u∈S') , p) ((v , v∈S') , q) =
    ((u · v · s') , SMultClosedSubset .multClosed (SMultClosedSubset .multClosed u∈S' v∈S') s'∈S')
    , path
   where
   path : u · v · s' · r · s'' ≡ u · v · s' · r'' · s
-  path = u · v · s' · r · s''   ≡⟨ cong (_· s'') (·-commAssocr _ _ _) ⟩
-         u · v · r · s' · s''   ≡⟨ cong (λ x → x · s' · s'') (·-commAssocr _ _ _) ⟩
-         u · r · v · s' · s''   ≡⟨ cong (_· s'') (·-commAssocr _ _ _) ⟩
+  path = u · v · s' · r · s''   ≡⟨ solve R' (X4 ·' X5 ·' X1 ·' X3 ·' X2)
```

Comment:
Here's an example how to use the solver @mzeuner 

This would of course be nicer with some reflection, but shortening these proofs using the explicit solver like this is still nice!

Reply:
Ah nice, it will be interesting to see how much we can simplify in that file.